### PR TITLE
8/soap/fixplugins

### PR DESCRIPTION
**Pull request by waddahadel**
I removed some test regarding the assignUser() method, the new signature returns void, which is ill typed here, the test is already redundant so it doesn't matter if we omit it, it won't hurt the logic

### CODE REVIEW

File: Services/Init/classes/class.ilErrorHandling.php [hunk 1]
```diff
@@ -37,7 +37,7 @@
  */
 class ilErrorHandling extends PEAR
 {
-    protected ?RunInterface $whoops;
+    protected ?RunInterface $whoops = null;
```
Comment (mjansenDatabay) [inline Services/Init/classes/class.ilErrorHandling.php:40]: Why do we need to apply this initial assignment in the `Init` component?

File: webservice/soap/classes/class.ilSoapRBACAdministration.php [hunk 2]
```diff
@@ -114,15 +114,15 @@ public function addUserRoleEntry(string $sid, int $user_id, int $role_id)
         if (!$ilAccess->checkAccess('edit_permission', '', $obj_ref)) {
             return $this->raiseError('Check access failed. No permission to assign users', 'Server');
         }
-
+/* Removal of unnecessary check
```
Comment (mjansenDatabay) [inline webservice/soap/classes/class.ilSoapRBACAdministration.php:117]: Maybe we should just remove the code block.

**Review by mjansenDatabay — COMMENTED** [review]
Hi @waddahadel ,

thanks for the PR. I added some inline comments. I am primarily interested in the `ErrorHandling` change. Is this change really necessary to make SOAP plugins work, if #7307 gets merged?

Best regards,
Michael

**Comment by waddahadel** [comment]
hey @mjansenDatabay ,

I think @jeph864 can answer the first comment, as for the second one, it's not really something that prevents the web service from working, I think it only throws an error in the log, cause that this method returns void, it used to return true in previous versions, the reporter said that the entry is still made regardless though,,,

**Comment by waddahadel** [comment]
I will open a new pr, I seemed to have messed up with a branch that belongs to @jeph864 in our fork, you can reject this and take a look at the new one @mjansenDatabay

**Pull request closed without merging**